Assets/Code/Scripts/PlayerManagement/MouseLook.cs

```diff
@@ -1,20 +1,28 @@
+using Unity.VisualScripting;
 using UnityEngine;
 
 public class MouseLook : MonoBehaviour
 {
+    /// <summary>
+    /// Everything here is linked to how cursor input is processed and used.
+    /// Rotation, orientation, sensitivity handling and eventual added preferences such as inverted vertical axis.
+    /// </summary>
+
```

Comment:
> - I think that the "obsolete" files in Arts/Background can be removed, right?

I'll check if it can be removed. There should be my placeholder there for the menu, I'll update you once I've checked.

> - If I click on the settings, I can only open the gameplay settings (and not the audio ones, for instance). Is this a bug, or do you plan of implementing this in the future?

This is a bug which Alessio encountered last week, and since you're having the same issue I'll try and see what causes it. Thanks for the feedback.

> - I have tried to thick the invert option, and the mouse behaves the same way (if I move up, the camera moves up, if I move down, the camera moves down). Maybe I did not get the role of that option, but I expected different behaviors

It's entirely my fault. I merged and did not check if the inverted Y continued to work. I'll open a branch to debug as soon as possible.

Reply:
> Picky me: the summary of the class should be above the class definition ;)

Yeah, sorry. I'll edit it out in the next commits.

Reply:
I've checked if the Y inversion works, and it does on my end. Did you confirm your settings before playing? 

Also I think the Art folder is still needed, since everything about logos and backgrounds will be stored there. What I can do to tidy up these folders is to move the UI folder into the Art one.

Let me know if you still experience the missing Y inversion. 

Last, but not least, indeed the text is all messed up and button area is not working as intended. Still trying to figure out what it is.

Reply:
Okay, the menu should now be fixed. Inverted Y still works, tried multiple times. I'm going to resolve this conversation, if you still experience this issue let me know.

Reply:
No I do not experience it, I was probably wrong and it was working since before. Anyway, now also the other settings are working! I'll approve